app/api/helpers/export_helpers.py

```diff
@@ -221,13 +221,7 @@ def export_event_json(event_id, settings):
     shutil.make_archive(dir_path, 'zip', dir_path)
     dir_path = dir_path + ".zip"
 
-    storage_path = UPLOAD_PATHS['exports']['zip'].format(
-        event_id=event_id
-    )
-    uploaded_file = UploadedFile(dir_path, dir_path.rsplit('/', 1)[1])
-    storage_url = upload(uploaded_file, storage_path)
-
-    return storage_url
```

Comment:
Why?

Reply:
@iamareebjamal the file path is turning out to be `http://localhost/events/exports/4/zip/aUdMcHJTSl/event4.zip`. `localhost` shouldn't be a part of the `dir_path` here. But anyway, I will refactor this so that this is served over a route, similar to how admin translations was implemented.

Reply:
But you have removed parts like uploaded file and upload function. Why were they here and why won't removing them cause issues

Reply:
the port is not getting attached to localhost
instead of `http://localhost:5000/events/exports/4/zip/aUdMcHJTSl/event4.zip`, it's becoming 
`http://localhost/events/exports/4/zip/aUdMcHJTSl/event4.zip`

Reply:
That is a long standing issue. But my point is that why will something not break if you have removed these lines of code

Reply:
@iamareebjamal the function `export_event_json` is only being used here so I believe that it won't break anywhere else?

Reply:
What about this line `storage_url = upload(uploaded_file, storage_path)`

Reply:
I've kept the original implementation intact and extracted the actual file path to be served so that the request is processed & goes to the newly implemented route.